### PR TITLE
LOOP-3951: Bump iOS min version to 15.0

### PR DESCRIPTION
**Pull request by rickpasetto**
https://tidepool.atlassian.net/browse/LOOP-3951

### CODE REVIEW

File: Version.xcconfig [hunk 1]
```diff
@@ -7,8 +7,8 @@
 //
 
 // Version [DEFAULT]
-LOOP_MARKETING_VERSION = 2.3.0
-CURRENT_PROJECT_VERSION = 57
+LOOP_MARKETING_VERSION = 0.0.1
```
Comment (darinkrauss) [inline Version.xcconfig]: This will need approval from @ps2. I think when this is merged back to DIY it is going to cause all sorts of confusion.

**Review by darinkrauss — COMMENTED** [review]
At least one change needs approval from @ps2, so just leaving my comment.

File: Loop.xcconfig [hunk 2]
```diff
@@ -42,6 +42,9 @@ LOOP_PROVISIONING_PROFILE_SPECIFIER_STATUS_EXTENSION_RELEASE =
 LOOP_PROVISIONING_PROFILE_SPECIFIER_WATCHAPP_RELEASE =
 LOOP_PROVISIONING_PROFILE_SPECIFIER_WATCHAPP_EXTENSION_RELEASE =
 
+// Min iOS Version [DEFAULT]					
+IPHONEOS_DEPLOYMENT_TARGET = 14.1
+
```
Comment (nhamming) [inline Loop.xcconfig:47]: PR title says bumping to min version to 15.0. Curious if this should be 14.1?

File: Version.xcconfig (hunk 1 above)
Comment (ps2) [reply Version.xcconfig]: Yes, I think the default here should remain the DIY version, and we should continue using our overridden version. I also think keeping the CURRENT_PROJECT_VERSION is useful for those building for testflight to have a single place to change it easily.

File: Loop.xcconfig (hunk 2 above)
Comment (rickpasetto) [reply Loop.xcconfig:47]: Ah, sorry if this is not obvious: 14.1 is what the default min version is today.  This specifies that for the DIY builds, effectively, and it is overridden by the [LW PR](https://github.com/tidepool-org/LoopWorkspace/pull/748).

Comment (ps2) [reply Loop.xcconfig:47]: I assume this will just be used for DIY, and will be overridden in our builds at a higher level.

Comment (rickpasetto) [reply Loop.xcconfig:47]: @ps2: Exactly.

File: Version.xcconfig (hunk 1 above)
Comment (rickpasetto) [reply Version.xcconfig]: Right.  Ok I will revert this.

**Review by ps2 — CHANGES_REQUESTED** [review]
I think the Version.xcconfig changes should be reverted, and Tidepool Loop will continue to override those in its builds.

**Review by darinkrauss — APPROVED** [review]
LGTM!

**Comment by rickpasetto** [comment]
@ps2 I think in order to merge this you need to approve because you requested changes.

**Review by ps2 — APPROVED** [review]
Thanks!

**Pull request merged**